### DISCUSSION TITLE
Multivendor Marketplace Plugin

### DISCUSSION THREAD

**Discussion by adam-ducksandwillows** [vendurehq/vendure#895] [General]
Dear All,

as discussed on discord, there are at least several people that need in one way or another functionality of multivendor marketplace. Micheal have in deed pointed out that, challange would be to somehow determine the union part of our requirements.

Im quit new to open source scene so, if there is something that you think should be proceed differently, let me know. Nevertheless here is my proposition:

Let's have a call/zoom/meets and talk first a littlest what we understand behind this definition, and see how much of this is common in our cases. I'm willing to host such a meeting, and try to moderate discussion.

Lets try to face first challange, called time zone.  I could do this meeting as early as tomorrow 20:00 CET. If you could please let me know if it is fine for you, and if not propose another day/time. 

If there is already process designed for such an community driven plugins requirement gathering and scope definition - please let me know, I'm from corporate world - don't want to reinvent the wheel.

Yours,
Adam

**Comment by nbezalwar** [comment]
IMO a good starting point would be to see what Vendure channels offers you as out-of-the-box.
https://www.vendure.io/docs/developer-guide/channels/

I think the immediate challenge would be with aggregating the search across all channels as the current graphQL Search APIs take a single channel. One possibility is to implement a third party search and discovery across all channels using something like Elastic, Klevu or  Algolia. I'm happy to participate in the discussions.

**Comment by michaelbromley** [comment]
Tracking this here: https://github.com/vendure-ecommerce/vendure/issues/1329

Please add further discussion to that issue.

**Comment by jsola02** [comment]
Hello, I would like to know what happened with the initiative or if there is something similar for multiple vendors within the same site?

**Reply by michaelbromley** [reply]
Hi, check the linked issue, #1329. I recently started working on this in earnest. Plan is that it is part of v2.0 in Q1 2023.
